pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/ExtractTransformFunctionTest.java

```diff
@@ -0,0 +1,44 @@
+package org.apache.pinot.core.operator.transform.function;
+
+import java.util.function.LongToIntFunction;
+import org.apache.pinot.common.function.scalar.DateTimeFunctions;
+import org.apache.pinot.common.request.context.ExpressionContext;
+import org.apache.pinot.common.request.context.RequestContextUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+
+public class ExtractTransformFunctionTest extends BaseTransformFunctionTest {
+
+  @DataProvider
+  public static Object[][] testCases() {
+    return new Object[][]{
+        {"year", (LongToIntFunction) DateTimeFunctions::year}, {"month",
```

Comment:
May I get eyes on this? This is after reformatting. IMO, it is not clearer.

Reply:
Yeah, we may use the `//@formatter:off` comment to skip the auto-formatting for this part. It looks cleaner previously.